spec/support/fixtures/test/main.swift

```diff
@@ -1,23 +1,23 @@
-protocol RedundantProtocol {
-}
-
-class SomeClass: RedundantProtocol {
-    enum SomeEnum {
-        case usedCase
-        case unusedCase
+    protocol RedundantProtocol {
     }
-
-    var unusedProperty = 0
-    private var assignOnlyProperty = 0
-
-
-    public func methodWithRedundantPublicAccessibility(_ unusedParameter: Int) {
-        assignOnlyProperty = 0
-        _ = SomeEnum.usedCase
+    
+    class SomeClass: RedundantProtocol {
+        enum SomeEnum {
+            case usedCase
+            case unusedCase
+        }
+    
+        var unusedProperty = 0
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/7e7bcf211941f63363d1ddade30bbc77d2b6c2c8/spec/support/fixtures/test/main.swift#L10"></span>Property 'unusedProperty' is unused</td>
    </tr>
  </tbody>
</table>